### PR TITLE
Create file_two

### PR DESCRIPTION
**Pull request by mikesokolsky**
More text

---
This change is [Reviewable](https://reviewable-dev-mvs.firebaseapp.com/reviews/mikesokolsky/pr-test-repo/2)

### CODE REVIEW

File: file_two [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ "three revolutionary products."
+
+These included a touch iPod, a better cellphone and a "breakthrough Internet communications device." Today we know 
+it simply as the iPhone.
+
+During the presentation, a group of Apple (AAPL, Tech30) employees who helped build the original iPhone sat in the
+audience bracing themselves for disaster.
+
+"The memory that I have from that day is doing shots of scotch that we had lined up because the demo was so tenuous," 
+Andy Grignon, a former Apple employee who was part of the original iPhone team, told CNNMoney. "It wasn't a sure thing."
```
Comment (mdevs5531) [inline file_two]: Really, CNNMoney quote?

File: file_two [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+After Jobs successfully demoed each feature, the lead designer took a victory shot of scotch.
+
+Scrolling through album covers? Shot. Pinch and zoom? Shot. Making a phone call? Shot, shot, shot.
+
+"It was this mix of absolute terror and elation," Grignon said.
+
+The presentation went off without a hitch and introduced the world to a product that would indeed make history. 
+A decade later, Apple has sold more than one billion iPhones. The device redefined the smartphone market, gave
```
Comment (mdevs5531) [inline file_two]: One billion dollars...

File: file_two (hunk 1 above)
Comment (mikesokolsky) [inline file_two]: Yep.

Comment (mdevs5531) [inline file_two]: A comment alone and isolated from a review.  Sad.

**Comment by mdevs5531** [comment]
A general comment.

File: file_two (hunk 1 above)
Comment (mikesokolsky) [inline file_two]: Not sure how to recover the link between comments :(

**Pull request merged**